docs/rfc/rfc-1.adoc

```diff
@@ -63,38 +63,41 @@ A user with a swept account balance can drain their account balance to redeem BT
 
 [[group-size]]
 === Group Size
-There are three main governable parameters that we need to choose:
 
-* The number of signers that can be controlled by the adversary before the key is in danger `threshold`
-* The number of group members required for a <<heartbeat,heartbeat>> to be considered "strong". `heartbeat`
-* The minimum number of members we're allowed to drop down to during the DKG
-  group formation re-try period. `dkg`
-* The total size of the group `groupSize`
+Governable Parameters:
+
+- `threshold`: The number of signers that can be controlled by the adversary
+  before the key is in danger.
+- `heartbeat`: The number of group members required for a
+  <<heartbeat,heartbeat>> to be successful.
+- `dkg`: The minimum number of members we're allowed to drop down to during the
+  DKG group formation re-try period.
+- `group_size`: The total size of the signing group.
```

Comment:
I am thinking whether `threshold` and `group_size` shouldn't be non-governable parameters requiring a hard-fork event to update. They should definitely be listed as parameters here, just unsure if governable. Or if governable, we could add a note that the update of these two parameters might be a hard-fork event (bridge component update).

Changing threshold/group size has a lot of implications for clients. For example, broadcast channel may be fine with 100 group members but not necessarily with 1000 group members.

Reply:
Can the contract allow governance to change them within hard-coded ranges? As in, we can change `group_size` from 50 to 150 and `threshold` from `group_size*.25` to `group_size*.75`?